Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -460,6 +466,15 @@ open class ChatMessageListVC:
             cell.configCell(messageCount: messagesCont)
             cell.transform = .mirrorY
             return cell
+        } else if channelType == .announcement {
+            let cell = listView.dequeueReusableCell(withIdentifier: "AnnouncementTableViewCell") as! AnnouncementTableViewCell
+            cell.btnContainer.addTarget(self, action: #selector(didSelectAnnouncement(_:)), for: .touchUpInside)
```

Comment:
@parth-simformsolutions  Let's not use outlets outside of its class.
And make these outlets as private.